### PR TITLE
TOF geometry update

### PR DESCRIPTION
**Pull request by yezhenyu2003**
TOF geometry updated (see issue #135) based on EIC Detector database tagged as 20220905113308.
Conflict with service cone to be resolved.

### CODE REVIEW

File: compact/tof_barrel.xml [hunk 1]
```diff
@@ -72,7 +72,7 @@
       <dimensions
         rmin="BarrelTOF_rmin-BarrelTOF_rOffset"
         rmax="BarrelTOF_rmax+BarrelTOF_rOffset"
-        length="BarrelTOF_length"/>
+	length="BarrelTOF_length"/>
```
Comment (wdconinc) [inline compact/tof_barrel.xml:75]: Please remove tab character.

File: compact/tof_endcap.xml [hunk 2]
```diff
@@ -39,8 +39,8 @@
     <comment> 1 um padding to not have layer and module touch (ACTS requirement) </comment>
     <constant name="TOFEndcapLayer_thickness"   value="TOFEndcapMod_thickness + 2 * TOFEndcapMod_dz + 1 * um" />
 
-    <constant name="ForwardTOFMod_rmin"         value="ForwardTOFRegion_tan * ForwardTOF_zmin" />
-    <constant name="ForwardTOFMod_rmax"         value="ForwardTOFRegion_rmax" />
+    <constant name="ForwardTOFMod_rmin"         value="12*cm" />
+    <constant name="ForwardTOFMod_rmax"         value="85*cm" />
```
Comment (wdconinc) [inline compact/tof_endcap.xml:43]: Please do not change parametrization with hard-coded values. If the values in the parametrization are incorrect, they should be adjusted there. @sly2j may comment.

File: compact/tof_endcap.xml [hunk 3]
```diff
@@ -51,8 +51,8 @@
     <constant name="ForwardTOFLayer1_zmin"      value="ForwardTOFMod1_zmin - TOFEndcapLayer_thickness / 2" />
     <constant name="ForwardTOFLayer2_zmin"      value="ForwardTOFMod2_zmin - TOFEndcapLayer_thickness / 2" />
 
-    <constant name="BackwardTOFMod_rmin"        value="BackwardPIDRegion_tan * BackwardTOF_zmin" />
-    <constant name="BackwardTOFMod_rmax"        value="BackwardPIDRegion_rmax" />
+    <constant name="BackwardTOFMod_rmin"        value="12*cm" />
+    <constant name="BackwardTOFMod_rmax"        value="64*cm" />
```
Comment (wdconinc) [inline compact/tof_endcap.xml:55]: Please do not change parametrization with hard-coded values.

File: compact/tof_endcap.xml [hunk 4]
```diff
@@ -7,13 +7,13 @@
...
+    <constant name="ForwardTOF_length"           value="15*cm"/>
+    <constant name="ForwardTOF_zmin"             value="163.0*cm"/>
+    <constant name="ForwardTOF_nlayers"          value="1"/>
     <constant name="ForwardTOFMod_offset"        value="1.2*cm"/>
-    <constant name="BackwardTOF_length"          value="min(BackwardTOFRegion_length, 2.*cm)"/>
-    <constant name="BackwardTOF_zmin"            value="BackwardTOFRegion_zmin + 0.5*cm"/>
-    <constant name="BackwardTOF_nlayers"         value="2"/>
+    <constant name="BackwardTOF_length"          value="10*cm"/>
+    <constant name="BackwardTOF_zmin"            value="166.0*cm"/>
+    <constant name="BackwardTOF_nlayers"         value="1"/>
```
Comment (wdconinc) [inline compact/tof_endcap.xml:16]: Please do not change parametrization with hard-coded values.

File: compact/tof_barrel.xml (hunk 1 above)
Comment (yezhenyu2003) [reply compact/tof_barrel.xml]: Done

File: compact/tof_endcap.xml (hunk 3 above)
Comment (yezhenyu2003) [reply compact/tof_endcap.xml]: Done

File: compact/tof_endcap.xml (hunk 4 above)
Comment (yezhenyu2003) [reply compact/tof_endcap.xml]: Done

**Comment by yezhenyu2003** [comment]
Thanks. What do I need to next?

Zhenyu

**Pull request merged**